modules/gapi/include/opencv2/gapi/rmat.hpp

```diff
@@ -54,11 +54,10 @@ class GAPI_EXPORTS RMat
     {
     public:
         using DestroyCallback = std::function<void()>;
+        using stepsT = std::vector<size_t>;
 
         View() = default;
-        View(const GMatDesc& desc, uchar* data, size_t step = 0u, DestroyCallback&& cb = nullptr)
-            : m_desc(desc), m_data(data), m_step(step == 0u ? elemSize()*cols() : step), m_cb(std::move(cb))
-        {}
+        View(const GMatDesc& desc, uchar* data, const stepsT& steps = {}, DestroyCallback&& cb = nullptr);
```

Comment:
So it breaks the compatibility with the existing code, isn't it?
Can this be done softer w/r/t the existing code? E.g. with a constructor overload.

Reply:
Yes, it breaks the compatibility, but only in one backend. The fix for this backend is ready so I propose not to leave a backward-compatible interface and merge the fix right after this PR is merged

Reply:
Added a backward compatible overload